js/bootstrap-slider.js

```diff
@@ -204,8 +204,26 @@
 			linear: {
 				toValue: function(percentage) {
 					var rawValue = percentage/100 * (this.options.max - this.options.min);
-					var value = this.options.min + Math.round(rawValue / this.options.step) * this.options.step;
+					if (this.options.ticks_positions) {
```

Comment:
@rovolution Thanks for the review. I tried that as well, but the small differences make it hard to capture it in a single method without severely hurting readability. I'd prefer to leave this as it is.